owncloudApp/src/main/res/xml/settings.xml

```diff
@@ -42,6 +43,19 @@
 
     </PreferenceCategory>
 
+    <PreferenceCategory
+        app:iconSpaceReserved="false"
+        app:key="logs_category"
+        app:title="@string/actionbar_logger">
+
+        <Preference
+            app:icon="@drawable/ic_baseline_description_24"
+            app:key="logs_subsection"
+            app:title="@string/prefs_enable_logging"
```

Comment:
Well it was a first approach, not sure about it either, but I chose it because it shares title with the first preference in the new screen `settings_logs.xml`. We'll let QA decide 😬 .